src/Core/BuildTools/Bind/ProfileWriter.cs

```diff
@@ -202,6 +200,8 @@ public static void WriteMixedModeClasses(this Project project, Profile profile,
             // }
             if (project.IsRoot)
             {
+                var partial = File.Exists(Path.Combine(folder, profile.ClassName + ".cs")) ? "partial" : "";
```

Comment:
Remove this. All classes should be partial in case we need to add custom functionality to them. Plus in this particular branch, this will always return an empty string as the empty mixed-mode class wouldn't have been produced yet in a clean build.